server/views/components/meta-unit/meta-unit.config.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import type {MetaUnit} from '../../../model/meta-unit';
+import {createMetaUnit} from '../../../model/meta-unit';
+
+export const name = 'meta-unit';
+export const handle = 'meta-unit';
+export const collated = true;
+
+export const metaUnit = createMetaUnit(({
+  type: 'text',
+  headingLevel: 'h3',
+  headingText: 'Curabitur quis',
+  content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur quis mollis turpis, eu facilisis ex. Vivamus sollicitudin pulvinar arcu sagittis posuere. Fusce a congue mauris. Vestibulum faucibus vel nibh ut commodo.',
+  includeDivider: true
+}: MetaUnit));
```

Comment:
We're using createMetaUnit and enforcing the type here, which is superfluous.
I personally prefer type casting i.e. `({/*stuff...*/}: MetaUnit)`, but either works.